deployment/ansible/roles/model-my-watershed.app/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
-- { include: configuration.yml }
 - { include: app.yml, when: "['packer'] | is_in(group_names)" }
+- { include: configuration.yml }
```

Comment:
This is so the working directory gets created before we try to configure `mmw.app.service`.

Reply:
Interesting. This makes sense because we enable and start the service via an Ansible task now.

Was this preventing the services from coming up on boot, or you just noticed the initial failures in the logs?

Reply:
This was preventing the services from coming up on boot, which caused instances to fail health checks.